File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/condicionais/q21.js
import { question } from "readline-sync";
function main(){
    //Input

    const num1 = Number(question("Enter a number value: "))
    const num2 = Number(question("Enter a second number value: "))

    //Processing

    const division_value = num1 / num2
    const rounded_division_value1 = division_value.toFixed(1)
    const rounded_division_value2 = division_value.toFixed(0)
    const remainder = rounded_division_value1 - rounded_division_value2

    if(remainder >= 0.5){
        const rounded_division_value2 = rounded_division_value2 + 1
        console.log(`The rounded value of the division between {num1} and {num2} is equal to: {rounded_division_value2}`)
    }
    else{
        console.log(`The rounded value of the division between {num1} and {num2} is equal to: {rounded_division_value2}`)
    }

    //Output
}

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/condicionais/q21.py
def main():
    #Input

    num1 = int(input("Enter a number: "))
    num2 = int(input("Enter a second number: "))

    #Processing

    division = num1 / num2
    division_rounded1 = round(division, 1)
    division_rounded = round(division)
    remainder = division_rounded1 - division_rounded

    if remainder >= 0.5:
        new_divison_round = division_rounded + 1
        print(f"The rounded number by the division between {num1} and {num2} is equal to: {new_divison_round}")
    else:
        print(f"The rounded number by the division between {num1} and {num2} is equal to: {division_rounded}")

    #Output

main()